backstop/tests/menu.html

```diff
@@ -493,7 +493,7 @@ <h3>Header</h3>
               ></svg-model>
               <span class="iui-button-label">
                 <div>Model Beta</div>
```

Comment:
Does this need to be changed to have a class name of `iui-header-breadcrumbs-button-name` like the Project Button example?